<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>–û—Å–Ω–æ–≤–Ω—ã–µ –ø–æ–ª–æ–∂–µ–Ω–∏—è –ø–æ –æ–ø—Ä–µ–¥–µ–ª–µ–Ω–∏—é —Ä–∞–∑–º–µ—Ä–æ–≤ —Å–≤—è–∑–µ–π</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>–†–∞—Å—á–µ—Ç–Ω—ã–π –Ω–æ—Ä–º–∞—Ç–∏–≤–Ω—ã–π –ø—Ä–µ–¥–µ–ª
 —Ç–µ–∫—É—á–µ—Å—Ç–∏ –ø–æ –Ω–æ—Ä–º–∞–ª—å–Ω—ã–º
–Ω–∞–ø—Ä—è–∂–µ–Ω–∏—è–º (–ú–ü–∞)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>–†–∞—Å—á–µ—Ç–Ω—ã–π –Ω–æ—Ä–º–∞—Ç–∏–≤–Ω—ã–π –ø—Ä–µ–¥–µ–ª
 —Ç–µ–∫—É—á–µ—Å—Ç–∏ –ø–æ –∫–∞—Å–∞—Ç–µ–ª—å–Ω—ã–º
–Ω–∞–ø—Ä—è–∂–µ–Ω–∏—è–º (–ú–ü–∞)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="lb_flowNormalStrain">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>0.00</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="lb_flowShearStrain">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>0.00</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>–ù–æ—Ä–º–∞–ª—å–Ω–æ–µ –Ω–∞–ø—Ä—è–∂–µ–Ω–∏–µ:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>–ö–∞—Å–∞—Ç–µ–ª—å–Ω–æ–µ  –Ω–∞–ø—Ä—è–∂–µ–Ω–∏–µ:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="lb_normalStrain">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>0.00</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="lb_shearStrain">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>0.00</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>–í–µ—Ä—Ö–Ω–∏–π –ø—Ä–µ–¥–µ–ª —Ç–µ–∫—É—á–µ—Å—Ç–∏: (–ú–ü–∞)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cbox_ReH">
            <item>
             <property name="text">
              <string>–í—ã–±–µ—Ä–µ—Ç–µ</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>235</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>315</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>355</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>&gt;=390</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>K–æ—ç—Ñ—Ñ–∏—Ü–∏–µ–Ω—Ç –∏—Å–ø–æ–ª—å–∑–æ–≤–∞–Ω–∏—è
 –º–µ—Ö–∞–Ω–∏—á–µ—Å–∫–∏—Ö —Å–≤–æ–π—Å—Ç–≤ —Å—Ç–∞–ª–∏</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="sb_coefficientNu"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>–£—á–µ—Ç –∫–æ—Ä—Ä–æ–∑–∏–æ–Ω–Ω–æ–≥–æ –∏–∑–Ω–æ—Å–∞</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>–ó–∞–ø–∞—Å –Ω–∞ –∏–∑–Ω–æ—Å ‚àÜùë†, –º–º, –ø—Ä–∏–Ω–∏–º–∞–µ—Ç—Å—è –¥–ª—è –∫–æ–Ω—Å—Ç—Ä—É–∫—Ü–∏–π, –ø–ª–∞–Ω–∏—Ä—É–µ–º—ã–π —Å—Ä–æ–∫ —Å–ª—É–∂–±—ã –∫–æ—Ç–æ—Ä—ã—Ö –ø—Ä–µ–≤—ã—à–∞–µ—Ç 12 –ª–µ—Ç</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QFrame" name="frame_3">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <spacer name="verticalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QFormLayout" name="formLayout_6">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_11">
                    <property name="font">
                     <font>
                      <pointsize>12</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>–ó–∞–ø–∞—Å –Ω–∞ –∏–∑–Ω–æ—Å:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLabel" name="lb_safetyMargin">
                    <property name="font">
                     <font>
                      <pointsize>12</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>0.00</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QFormLayout" name="formLayout_7">
               <item row="0" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>C—Ä–µ–¥–Ω–µ–≥–æ–¥–æ–≤–æ–µ —É–º–µ–Ω—å—à–µ–Ω–∏–µ —Ç–æ–ª—â–∏–Ω—ã —Å–≤—è–∑–∏, –º–º/–≥–æ–¥,
 –≤—Å–ª–µ–¥—Å—Ç–≤–∏–µ –∫–æ—Ä—Ä–æ–∑–∏–æ–Ω–Ω–æ–≥–æ –∏–∑–Ω–æ—Å–∞ –∏–ª–∏ –∏—Å—Ç–∏—Ä–∞–Ω–∏—è</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="le_thicknessElem"/>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="le_serviceLife"/>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>–ü–ª–∞–Ω–∏—Ä—É–µ–º—ã–π —Å—Ä–æ–∫ —Å–ª—É–∂–±—ã –∫–æ–Ω—Å—Ç—Ä—É–∫—Ü–∏–∏ (–≥–æ–¥—ã)</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="pb_caiculus">
                 <property name="text">
                  <string>–†–∞—Å—á—ë—Ç</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <layout class="QFormLayout" name="formLayout_5">
             <item row="0" column="0">
              <widget class="QLabel" name="label_12">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>–ö–æ—ç—Ñ—Ñ–∏—Ü–∏–µ–Ω—Ç œâ–∫, —É—á–∏—Ç—ã–≤–∞—é—â–∏–π
 –ø–æ–ø—Ä–∞–≤–∫—É –Ω–∞ –∏–∑–Ω–æ—Å</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="lb_correctionForWear">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>0.00</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout_4">
             <item row="0" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>M–æ–º–µ–Ω—Ç —Å–æ–ø—Ä–æ—Ç–∏–≤–ª–µ–Ω–∏—è —Ä–∞—Å—Å–º–∞—Ç—Ä–∏–≤–∞–µ–º–æ–π –±–∞–ª–∫–∏</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="le_beamResistance"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
